web/views.py

```
@@ -3110,24 +3110,30 @@ def system_status(request):
         message = request.POST.get("message")
 
         if title and message:
+            # Import the encryption function from our secure messaging module
+            from web.secure_messaging import encrypt_message
+
+            # Encrypt the message using Fernet and decode to string for email content
+            encrypted_message = encrypt_message(message).decode("utf-8")
+
             # Get all enrolled students
             enrolled_students = User.objects.filter(
                 enrollments__course=course, enrollments__status="approved"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)